docs/quick-starts/bandit-quickstart.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: /bandit-quickstart
+sidebar_position: 5
+---
+
+# Running your first Contextual Bandit
+
+This 10 minute guide will walk through configuring and running your first Contextual Bandit with Eppo. Note that Contextual Bandits leverage the Eppo SDK and this guide will walk through SDK setup.
+
+:::info
+In order to optimize a Bandit, you must have [created your first metric](/metric-quickstart).
+:::
+
+### 1. Generate an SDK key
+
+From the Configuration section, navigate to the SDK keys tab. Here you can generate keys for both production and testing.
+
+![Setup Eppo SDK key](/img/feature-flagging/environments/sdk-keys.png)
+
+For now, create a Test environment SDK key by using the "New SDK Key" button. Give the key a name and select "Test" for the Environment.
+
+![Generate a SDK key](/img/feature-flagging/sdk-key-modal.png)
+
+Store the SDK key securely; it is not possible to view it after closing the modal. However, generating a new key is easy in case you do lose it.
+
+### 2. Connect a logging function to the Eppo SDK
+
+Eppo leverages your existing event logging infrastructure to track experiment assignment. Whether you are using a third party system to log events to the data warehouse or have an internally built solution, you'll simply pass in a logging function when initializing the SDK.
+
+The [event logging](/sdks/event-logging/) page has more information on how to set up logging using different logging tools
+
+This logger should write to a table with the following columns (they can be in any order):
```

Comment:
Assuming the names have to to exactly those:
```suggestion
This logger should write to a table with columns with the following names (they can be in any order):
```
Otherwise, 
```suggestion
This logger should write to a table with columns with the following information. They can be in any order and have different names, but you will need to configure them to let Eppo know which is which:
```